recipe/build.sh

```diff
@@ -2,14 +2,18 @@
 
 set -e
 
-test -d m4 || mkdir m4
-# gtkdocize || exit 1
+echo "DIAGNOSTIC environment:"
+env | sort
 
-export ACLOCAL_PATH="$PREFIX/share/aclocal"
-aclocal --print-ac-dir
+echo "DIAGNOSTIC directory listing:"
+ls -lAFgh
 
-autoreconf -ivf
+echo "DIAGNOSTIC pwd: $(pwd)"
 
-./configure --disable-static --disable-gui --enable-introspection=yes --disable-hkl-doc --prefix=$PREFIX
-make -j ${CPU_COUNT}
-make install
+TARBALL="libhkl-${PKG_VERSION}-x86_64.tar.gz"
+echo "Installing from ${TARBALL}"
+tar xzf "${TARBALL}" -C "${PREFIX}"
```

Comment:
So, the approach is to not build it here with conda, but use the pre-built archive?
I wonder if the approach may have some consequences if the toolset in conda-forge changes. Any thoughts?

Reply:
I believe we can respond more easily if the toolset in conda-forge changes.  Other C library packages in conda-forge also build a tarball outside of the recipe, as done here.

Reply:
The way we respond is to build those changes into the tarball process, then re-release a new tarball as needed.